hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/LruCachedBlockQueue.java

```diff
@@ -20,6 +20,7 @@
 import org.apache.hadoop.hbase.io.HeapSize;
 import org.apache.yetus.audience.InterfaceAudience;
 
+import org.apache.hbase.thirdparty.com.google.common.base.Preconditions;
```

Comment:
All the changes in this file seem extraneous?
Perhaps the whitespace change will avoid a spotless warning and findbugs exclusion can make a difference in the precommit report, but overall these changes come from somewhere other than the thirdparty changes?

Reply:
These changes are also part of HBASE-26523 backport [here](https://github.com/apache/hbase/commit/e53712ae993248594245363fc3e8e36022534d9a#diff-b73c64fdb993ed6467649561274720735b666ddd05a768aa982766e9a9371d19R22-R25)

Reply:
Since it is not a merge issue, fine